File: Array/Reverse.py
def reverseArray(arr):
    start = 0
    end = len(arr) - 1
    while start < end:
        temp = arr[start]
        arr[start] = arr[end]
        arr[end] = temp
        start += 1
        end -= 1


n = int(input('Enter size of Array: '))
print(f'Enter {n} elements:')
arr = list(map(int, input().split()))
reverseArray(arr)
print('Reversed Array is: ')
print(arr)

File: Array/Reverse.java
package Array;

public class Reverse {

  public static int[] reverseArray(int[] arr, int size) {
    int left = 0, right = size - 1;
    while (left < right) {
      int temp = arr[left];
      arr[left] = arr[right];
      arr[right] = temp;
      left++;
      right--;
    }
    return arr;
  }
  public static void main(String[] args) {
    int[] arr = Basic.inputArray();
    int[] res = reverseArray(arr, arr.length);
    System.out.println("Reversed Array is: ");
    Basic.printArray(res);
  }
}
